tsconfig.json

```diff
@@ -1,10 +1,10 @@
 {
-  "strict": true,
```

Comment:
`tsup` was flagging this because `strict` should be within the `compilerOptions` object:

![CleanShot 2024-08-01 at 09 47 20@2x](https://github.com/user-attachments/assets/6eca9d8b-f6d6-472d-8a7e-cd1850f9a413)